client/src/lifecycleManager.js

```diff
@@ -29,16 +29,18 @@ class LifecycleManager extends LuigiClientBase {
 
     /**
      * Iterates over an object and executes all top-level functions
-     * with a given payload.
+     * with a given payload and the Luigi core domain
      * @private
      */
-    const _callAllFns = (objWithFns, payload) => {
+    const _callAllFns = (objWithFns, ...names) => {
```

Comment:
There is one call to this function which is not sending the origin (`luigi.navigate` listener). Anyway what about calling here `Helpers.getLuigiCoreDomain()` instead of having the origin as input parameter?

Reply:
talked with Philipp, we don't need this in luigi.navigate.

Reply:
Ok cool. Is there any special reason why we are we using "...names" here? Maybe we could use `(objWithFns, payload, luigiCoreOrigin)`

Reply:
good point